services/scheduler/src/test/java/org/mule/service/scheduler/internal/DefaultSchedulerQuartzTestCase.java

```diff
@@ -54,7 +54,8 @@
   public void before() throws SchedulerException {
     super.before();
     executor = new DefaultScheduler(DefaultSchedulerQuartzTestCase.class.getSimpleName(), sharedExecutor, 1,
-                                    sharedScheduledExecutor, sharedQuartzScheduler, CUSTOM);
+                                    sharedScheduledExecutor, sharedQuartzScheduler, CUSTOM, sched -> {
```

Comment:
Crate `null` consumer or SchedulerStopper for clarity instead of repeating empty consumer in multiple places?